x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list.tsx

```diff
@@ -72,16 +73,51 @@ const MAX_RESULTS = 1000;
 
 const ruleEventListContainerStyle = { minHeight: 400 };
 
-export type RuleEventLogListProps = {
+export type RuleEventLogListOptions = 'stackManagement' | 'default';
+
+export interface RuleEventLogListCommonProps {
   rule: Rule;
+  ruleType: RuleType;
   localStorageKey?: string;
   refreshToken?: number;
   requestRefresh?: () => Promise<void>;
   customLoadExecutionLogAggregations?: RuleApis['loadExecutionLogAggregations'];
-} & Pick<RuleApis, 'loadExecutionLogAggregations'>;
-
-export const RuleEventLogList = (props: RuleEventLogListProps) => {
-  const { rule, localStorageKey = RULE_EVENT_LOG_LIST_STORAGE_KEY, refreshToken } = props;
+  loadExecutionLogAggregations?: RuleApis['loadExecutionLogAggregations'];
```

Comment:
nit -> I will love if we can remove this two attributes `customLoadExecutionLogAggregations` and `loadExecutionLogAggregations`. They always confusing me ;)

Reply:
removed `customLoadExecutionLogAggregations`, still need `loadExecutionLogAggregations` since that's the rule API that gets injected with the HOC